src/packages/board/board.py

```diff
@@ -0,0 +1,206 @@
+"""
+board.py - rescue board tracking facility
+
+
+Copyright (c) 2019 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+"""
+from __future__ import annotations
+
+import itertools
+import logging
+import typing
+from collections import abc
+from contextlib import asynccontextmanager
+from uuid import UUID
+
+from src.packages.rescue import Rescue
+
+CYCLE_AT = 15
```

Comment:
This should be a configurable value, not hard coded, especially if it isn't private.

Reply:
Hm, I don't really foresee a need to increase this limit in a configurable fashion, after all its meerly a hint for when mecha should attempt to start over indexing.

Reply:
made it configurable. in ebdf85a